### DISCUSSION TITLE
Updating fields in DOCX doesn't work in latest version?

### DISCUSSION THREAD

**Discussion by rvanaalderen** [gotenberg/gotenberg#1060] [Q&A]
We switched from version gotenberg:7.1.1 to 8.13 and we find out that it handles updating the ‘fields’ differently.

In MS Word this popup is a shown.
![image](https://github.com/user-attachments/assets/11c2142d-0042-438d-9885-a2bd166fb68d)

In 7.1.1 the fields were updated by default and in version 8.13 it seems that this is not done. 
Have any default settings been changed between version 7.1.1. and 8.13

**Comment by gulien** [comment]
Nothing I am aware of. It could be an update in LibreOffice directly 🤷
